### DISCUSSION TITLE
What's the password for the testing GraphQL server?

### DISCUSSION THREAD

**Discussion by yf-hk** [refinedev/refine#1842] [Q&A] [answered]
Hi all,
I was new to Refine, trying to figure out how to use it with Strapi GraphQL data provider. After initializing the project with `superplate-cli` it generated a `gqDataProvider.ts` file, I can see that the endpoint was set to `https://api.strapi.refine.dev/graphql`. After adding resourses `resources={[{ name: 'posts' }]}`, I can see a login page like this:

![image](https://user-images.githubusercontent.com/203980/170595674-cde2d1e7-567a-496d-9c6b-4c9fbb14f2f5.png)

I wonder is this a fully functioning Strapi GraphQL test backend? What's the correct username and password if I would like to login?

Many thanks!

**Comment by yf-hk** [comment]
To answer my own question, the login seems to be `demo@refine.dev`, `demodemo`. But I can't get it work.

![image](https://user-images.githubusercontent.com/203980/170604789-8ef1d0e0-f3c3-45d1-bee3-47329a2d0f1d.png)

**Comment by omeraplak** [comment, marked as answer]
Hey @dzcpy ,
Thank you for contacting us! Sorry for the delay, we forgot to reply :(

You can check this example for the Strapi GraphQL,

https://refine.dev/docs/examples/data-provider/strapi-graphql/
